lib/src/layout/grid_layout.dart

```diff
@@ -144,3 +163,19 @@ class _GridLayoutState extends State<GridLayout> {
     );
   }
 }
+
+class DisabledVideoStfWidget extends StatefulWidget {
```

Comment:
@tadaspetra Because `_getRenderViews()` creates a list of stateful widgets for all the user views. So to add a disabled video widget into that list it needs to be stateful too.